Lib/fontTools/cffLib.py

```diff
@@ -1484,37 +2030,17 @@ def getChildren(self, strings):
 		return children
 
 
-class FontDictCompiler(TopDictCompiler):
+class FontDictCompiler(DictCompiler):
```

Comment:
Here it seems like you reverted the patch #744:

https://github.com/fonttools/fonttools/pull/744/files#diff-5c4ae0351e8d367dbe256f28eba7db01R1487

@justvanrossum added that to fix issue #740.

@readroberts Did you do that intentionally?
Or perhaps you based your CFF2 patch on an outdated master?